zbark/debark.go

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package zbark
+
+import (
+	"github.com/uber-go/zap"
+	"github.com/uber-go/zap/zwrap"
+
+	"github.com/uber-common/bark"
+)
+
+// Debarkify wraps bark.Logger to make it compatible with zap's JSON logger
+func Debarkify(b bark.Logger) zap.Logger {
+	if wrapper, ok := b.(*barker); ok {
+		return wrapper.zl
+	}
+	return &zapper{
+		lvl: zap.DebugLevel,
```

Comment:
rather than default to debug, should we take level as an argument to Debarkify? I'm not 100% sure how we want zap's levels to interact with the logrus loggers